BrainPortal/app/models/cluster_task.rb

```diff
@@ -2372,6 +2372,8 @@ def load_singularity_image_from_repo #:nodoc:
     errfile = "/tmp/.container_load_cmd.#{self.run_id}.err"
     success = system("#{singularity_executable_name} pull --name #{image_name.bash_escape} #{singularity_image_name.bash_escape} </dev/null >/dev/null 2>#{errfile.bash_escape}")
     err     = File.read(errfile) rescue "No Error File?"
+    # Singularity command can generate `implausibly old timestamp` when pulling a docker image (due to tar), we ignore it. 
+    err.gsub!(/^.implausibly old time stamp.$\n?/,"")
```

Comment:
Is the real message with `timestamp` or `time stamp` ?!? The comment and code disagree.